### DISCUSSION TITLE
Disabling certificate validation

### DISCUSSION THREAD

**Discussion by markhaehnel** [bytebeamio/rumqtt#757] [Q&A]
Hi everyone,
is there currently a way to disable certificate validation on the client using either rustls or native-tls?
I'm connecting to a broker that only serves over tls connection which has a self-signed cert but don't want to add it's cert to my system's cert store.

Thank you in advance!

**Comment by de-sh** [comment]
Hi @markhaehnel sorry for the late reply, you can configure [tls](https://docs.rs/rumqttc/latest/rumqttc/enum.TlsConfiguration.html) as follows:
```rs
TlsConfiguration::SimpleNative {
    ca: "...<CA certificate body>...".to_vec(),
    client_auth: None,
}
```

**Reply by markhaehnel** [reply]
Thanks for the reply!
This means i need to know the certificate in advance and cannot just ignore it, right?
